app/Http/Controllers/Admin/UsersController.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Http\Requests;
+use Auth;
+use Notifications;
+
+class UsersController extends Controller
+{
+    private $user;
+
+    public function __construct()
+    {
+        \Title::prepend(trans('dashboard.title.prepend'));
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(User $user)
+    {
+        $admins = $user->orderBy('updated_at', 'desc')->paginate(20);
+
+        \Title::append(trans('users.title.index'));
+
+        return view('admin.users.index', ['user' => Auth::user(), 'admins' => $admins]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        \Title::append(trans('users.title.create'));
+
+        return view('admin.users.create', ['user' => Auth::user()]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Requests\AdministratorCreationRequest $errors, User $user)
+    {
+        $user->create([
+            'name' => request('name'),
+            'email' => request('email'),
+            'password' => request('password'),
+        ]);
+
+        Notifications::success(trans('users.notification.user-created-success'), 'top');
+
+        return redirect()->route('admin.users.index');
```

Comment:
Please, let's be consistent.
use `admin-users-index`.
I mean, let's separate it with dash symbol. And methods like `post`, `put`, `delete` separate with `.`.
Example:
`admin-users-edit`
`admin-users-edit.put`